Exec/unit_tests/test_basestate/MaestroInit.cpp

```diff
@@ -104,22 +104,6 @@ void Maestro::InitData() {
     // set p0^{-1} = p0_old
     p0_nm1.copy(p0_old);
 
-    rhoX0_old.resize(base_geom.max_radial_level + 1, base_geom.nr_fine,
-                     NumSpec);
```

Comment:
why was this removed?

Reply:
I removed `rhoX0_old` and `rhoX0_new` entirely because they were only used by this test problem and are not needed